### PR TITLE
Test import & proxy for napalm_network module

### PR DESCRIPTION
**Pull request by mirceaulinic**
### What does this PR do?

Will test if the NAPALM library is installed and the proxy module is used.

### CODE REVIEW

**Comment by cro** [comment]
Hey @mirceaulinic -- will get this merged as soon as Jenkins recovers.

File: salt/modules/napalm_network.py [hunk 1]
```diff
@@ -41,7 +52,17 @@
 
 
 def __virtual__():
-    return True
+
+    """
```
Comment (rallytime) [inline salt/modules/napalm_network.py]: These should be single quotes instead of double quotes here, too.

**Comment by rallytime** [comment]
@mirceaulinic It looks like there is a [lint error](http://166.78.178.63:8080/job/PR/job/salt-pr-lint-n/1325/violations/file/salt/modules/napalm_network.py/) here. Mind fixing that up?

**Comment by mirceaulinic** [comment]
After I replaced the quotes, I forgot to check for lint errors. Should be fixed now.

**Comment by rallytime** [comment]
@mirceaulinic Sorry for the trouble here, but it looks like this isn't quite right still.

**Comment by mirceaulinic** [comment]
My bad :(

**Comment by rallytime** [comment]
Thanks @mirceaulinic!

**Pull request merged**